Xamarin.Forms.Core/AbsoluteLayout.cs

```diff
@@ -225,10 +225,10 @@ static Rectangle ComputeLayoutForRegion(View view, Size region)
 			SizeRequest sizeRequest;
 			Rectangle bounds = GetLayoutBounds(view);
 			AbsoluteLayoutFlags absFlags = GetLayoutFlags(view);
-			bool widthIsProportional = (absFlags & AbsoluteLayoutFlags.WidthProportional) != 0;
-			bool heightIsProportional = (absFlags & AbsoluteLayoutFlags.HeightProportional) != 0;
-			bool xIsProportional = (absFlags & AbsoluteLayoutFlags.XProportional) != 0;
-			bool yIsProportional = (absFlags & AbsoluteLayoutFlags.YProportional) != 0;
+			bool widthIsProportional = absFlags.HasFlag(AbsoluteLayoutFlags.WidthProportional);
```

Comment:
We intentionally avoid the `HasFlag` extension method because it is much slower.